### PR TITLE
update d2l-video

### PR DESCRIPTION
**Pull request by imcdonald-d2l**
Update `d2l-video` version. Updated sequence version and BSI `d2l-video` update to follow merge so it works locally.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -63,10 +63,10 @@
   },
   "main": "d2l-sequences.js",
   "dependencies": {
-    "@brightspace-ui/core": "^1",
+    "@brightspace-ui/core": "^1.53.0",
```
Comment (ianlaidlaw) [inline package.json:66]: same question here, any reason to move away from `"^1"` ?

Comment (imcdonald-d2l) [reply package.json:66]: I (improperly) assumed it needed the version updated. See comment on lessons PR for more context. Will update this as well

File: .gitignore [hunk 2]
```diff
@@ -2,5 +2,4 @@ node_modules
 locales.json
 .vs
 analysis.json
-package-lock.json
```
Comment (ChrisKraljevicD2L) [inline .gitignore:5]: Is there a reason why `package-lock` is being reinstated here?

Comment (imcdonald-d2l) [reply .gitignore:5]: Travis build wouldn't pass without it for some reason. Not sure if there is a way around this 🤷‍♂️

**Pull request merged**